### DISCUSSION TITLE
UI Redisign Mockup

### DISCUSSION THREAD

**Discussion by daudix** [GradienceTeam/Gradience#25] [Ideas]
Hi, I have made this mockup with idea to keep current UI as much as possible, but in same time, make it new, **any suggestions appreciated!**

_New welcome screen, preset selector and full theme preview will be implemented (hopefully) in V2_

# V1

![Adwaita Manager V1](https://user-images.githubusercontent.com/77155297/181392263-97eab96f-d386-4d12-95bb-0571b3411796.png)

# V2

![Adwaita Manager V2](https://user-images.githubusercontent.com/77155297/181852906-4ab61ab7-7a86-4c72-9d22-e4a6d0c27312.png)

_[📎 SVG, PNG & All Future Versions](https://drive.google.com/drive/folders/1a7SDgbVaGE9GnsEW2udspE6YyXrFrNnK?usp=sharing)_

**Comment by daudix** [comment]
About Monet (Material You - Material 3) Engine, there is a few options, one of this is [C# Implementation](https://github.com/albi005/MaterialColorUtilities) of Monet used in [KDE Material You Colors](https://github.com/luisbocanegra/kde-material-you-colors) (Written with Python), Maybe author of this tool will help with this

_BTW, [u/aviroblox](https://www.reddit.com/user/aviroblox/) on Reddit done this manually using Adwaita Manager, [Here is it](https://www.reddit.com/r/gnome/comments/w8yvjp/copied_my_materialu_theme_to_gnome_applications/)_

**Comment by daudix** [comment]
@avanishsubbiah Have made script for this in #27

**Comment by ArtyIF** [comment]
Where would the preset selector go?

**Reply by daudix** [reply]
I don't know where to place it, any ideas?

**Reply by daudix** [reply]
Done, will add V2 soon

**Comment by daudix** [comment]
@ArtyIF V2 Released, with Preset Selector and a lot more

**Reply by ArtyIF** [reply]
I have a question - did you design the presets icon yourself or did it come from the icon development kit? Because I can't find it anywhere.

**Reply by daudix** [reply]
any icons? most of them are form icon library (but not all)

**Comment by ArtyIF** [comment]
What's the name of the icon on the presets dropdown? With the 
fan-looking thing.

**Reply by daudix** [reply]
It's not aviable in icon library, instead, it come from Palette app, here all custom icons saved as symbolics.

_[📎 CustomSymbolics.zip](https://github.com/ArtyIF/AdwCustomizer/files/9235775/CustomSymbolics.zip)_

**Reply by ArtyIF** [reply]
[This Palette app](https://apps.gnome.org/app/org.gnome.design.Palette/)?

**Reply by daudix** [reply]
Exactly.

**Reply by ArtyIF** [reply]
The issue I have is that I'm not sure how to credit the original designer, Tobias Bernard, and the licensing - Palette uses GPLv3, while I use MIT/X11.

**Reply by ArtyIF** [reply]
Using a different icon right now

**Reply by daudix** [reply]
Just link to Tobias Bernard [Website](https://tobiasbernard.com/), but I'm don't sure about licencing, maybe change our license to GPLv3?

**Reply by daudix** [reply]
> Using a different icon right now

Let me guess, this one?

![image](https://user-images.githubusercontent.com/77155297/182205306-808ec925-1dba-4c0b-94e4-d890aa460830.png)

**Comment by ArtyIF** [comment]
No. I did at first, but since it's the presets icon I changed it to the 
"controls" icon.

**Reply by daudix** [reply]
Good choise, but also I make similar "Palette" icon.

**Reply by daudix** [reply]
Here we go!

![image](https://user-images.githubusercontent.com/77155297/182211878-4d95cc07-efaa-4eda-b8dc-0bb7aa3db8e1.png)

_[📎 palette-symbolic.zip](https://github.com/ArtyIF/AdwCustomizer/files/9236160/palette-symbolic.zip)_

**Comment by daudix** [comment]
About licencing, is there any particular reason to use MIT over GPLv3?

**Comment by 0xMRTT** [comment]
Hi @daudix-UFO.

I think that all mockups can go in a separate repository?

**Reply by daudix** [reply]
Like repository only for mockups? can we make one?

**Reply by ArtyIF** [reply]
> Like repository only for mockups? can we make one?

Yeah, similar to how GNOME does it. I think you should have rights to make a new repository in an organization.

**Reply by 0xMRTT** [reply]
Yes, created one called design

**Reply by daudix** [reply]
Amazing, will move mockups to it! and also make V3 with plugins (modules?) support!

**Comment by 0xMRTT** [comment]
Mockups moved to the Design repository
